README.md

```diff
@@ -199,19 +259,24 @@ the Functions Framework:
 --target com.example.HelloWorld
```

Comment:
I think it would make sense to delete these four lines in favour of the bullet list you've introduced, as in the next section.